tests/venv/repo-path/verify

```diff
@@ -7,4 +7,8 @@ assert sys.executable == '/srv/conda/bin/python'
 
 # Repo should be in /srv/repo
 assert os.path.exists('/srv/repo/verify')
-assert os.path.abspath(__file__) == '/srv/repo/verify'
\ No newline at end of file
+assert os.path.abspath(__file__) == '/srv/repo/verify'
+
+# Verify that ~/.conda is owned by current user, and not root
+assert os.stat(os.path.expanduser('~/.conda')).st_uid == 1000
```

Comment:
This should probably use `os.environ['NB_UID']` since the tests aren't always run as uid 1000 (it's the calling user's uid by default, I think)